packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -1689,28 +1689,76 @@ void main() {
       // Hold the press.
       await tester.pump(const Duration(milliseconds: 500));
 
-      // The obscured text is not broken into words, so only one letter is
-      // selected at a time.
+      // The obscured text is treated as one word, should select all
       expect(
         controller.selection,
-        const TextSelection(baseOffset: 9, extentOffset: 10),
+        const TextSelection(baseOffset: 0, extentOffset: 35),
       );
 
       // Selected text shows 3 toolbar buttons.
-      expect(find.byType(CupertinoButton), findsNWidgets(3));
+      expect(find.byType(CupertinoButton), findsNWidgets(1));
```

Comment:
The comment says there should be 3, why did that change?

Reply:
it used to be copy, paste, and cut.
after this change it by default can only paste